### PR TITLE
[HCPCFS-3261] [Webhook Compliance] Fix issues identified from SF whitesource scan

### PR DESCRIPTION
**Pull request by jintusebastian**
Resolving issues in webhooks component found in Service Fabric whitesource scan:

- Updating following libraries:
      - github.com/golang/glog v1.0.0
      - github.com/google/uuid v1.3.0
      - github.com/onsi/ginkgo v1.16.5
      - github.com/onsi/gomega v1.17.0
      - k8s.io/api v0.23.3
      - k8s.io/apimachinery v0.23.3
      - k8s.io/client-go v0.23.3
- Migrating from Dep to Go Modules
- Updating travis build for webhooks
- Adding staticcheck for webhooks
- Updated aut-generated components

Related boshrelease cloudfoundry-incubator/service-fabrik-boshrelease/pull/231

### CODE REVIEW

**Comment by vinaybheri** [comment]
Validation on k8s cluster: n+n+n failed.

**Comment by vinaybheri** [comment]
Validation on k8s cluster: n-2 failed.

**Comment by vinaybheri** [comment]
Validation on k8s cluster: n-1 failed.

**Comment by vinaybheri** [comment]
Validation on k8s cluster: n-1 failed.

**Comment by vinaybheri** [comment]
Validation on k8s cluster: n-2 failed.

**Comment by vinaybheri** [comment]
Validation on k8s cluster: n+n+n failed.

**Comment by vinaybheri** [comment]
Validation on k8s cluster: n+n+n failed.

**Comment by vinaybheri** [comment]
Validation on k8s cluster: n-1 failed.

File: webhooks/pkg/webhooks/manager/webhook_test.go [hunk 1]
```diff
@@ -19,8 +19,6 @@ import (
 
 type MockEvent struct {
 	AdmissionReview      *v1beta1.AdmissionReview
-	crd                  resources.GenericResource
-	oldCrd               resources.GenericResource
```
Comment (jintusebastian) [inline webhooks/pkg/webhooks/manager/webhook_test.go:23]: > field crd is unused
> field oldCrd is unused

Caught in staticcheck

**Review by anoopjb — APPROVED** [review]
LGTM 👍

**Comment by vinaybheri** [comment]
Validation on k8s cluster: n is succeeded.

**Comment by vinaybheri** [comment]
Validation on k8s cluster: n-1 is succeeded.

**Comment by vinaybheri** [comment]
Validation on k8s cluster: n-2 is succeeded.

**Pull request merged**